roles/lnls-ans-role-update-kernel/tasks/update-kernel.yml

```diff
@@ -0,0 +1,6 @@
+---
+- name: Update kernel
+  package:
```

Comment:
Does using the `package`module works with the enabled repositories. I know we must have stretch-backports repository enable with this kernel, but I'm not sure if `package` module can do this.

Reply:
Also, when we install packages for Debian we always update the cache first and then proceed to install the packages. This could be important if we are running only this role, as it will not update the apt cache.

Reply:
> Does using the `package`module works with the enabled repositories? I know we must have stretch-backports repository enabled to install this kernel, but I'm not sure if `package` module can do this.

I tried a previous version of this PR in three hosts. It worked.

Reply:
Nice. Ok then

Reply:
> Also, when we install packages for Debian we always update the cache first and then proceed to install the packages. This could be important if we are running only this role, as it will not update the apt cache.

I see... done.